src/browser/perspective-ui.js

```diff
@@ -130,6 +134,10 @@ export const step = (model, action) => {
       const [browser, fx] = Browser.step(model.browser, action);
       return [asShowWebView(browser), fx];
     }
+    else if (action.target === 'Browser.KeyUp' && action.action.key === 'Escape') {
```

Comment:
Is this how we want to do this, or should I be using https://github.com/mozilla/browser.html/pull/714/files#diff-5c204f4fed6072a5044a37de83d15202R60? Except that `isEscape` doesn't seem to be defined or used anywhere. Is it actually a predicate function for detecting escape key?